website/server/models/user/schema.js

```diff
@@ -357,6 +358,7 @@ let schema = new Schema({
     party: {type: Schema.Types.Mixed, default: () => {
       return {};
     }},
+    parties: {type: [{}], default: () => []},
```

Comment:
We try to be as specific as possible with _most_ attributes. Helps us keep data in a known good state. So in this case, it would be:

```js
parties: [{
  id: {
    type: String, 
    ref: 'Group', 
    required: true,
    validate: [validator.isUUID, 'Invalid uuid.'],
  }, 
  name: {
    type: String, 
    required: true,
  }, 
  inviter: {
    type: String, 
    ref: 'User', 
    required: true,
    validate: [validator.isUUID, 'Invalid uuid.'],
  },
}],
```